lab13-jacob/index.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+let express = require('express');
+let mongoose = require('mongoose');
+let jsonParser = require('body-parser').json();
+
+
+let MONGODB_URI = process.env.MONGODB_URI || 'mongodb://jisenber:Ji52617!@ds141128.mlab.com:41128/isenbergdb';
+let PORT = process.env.PORT || 3000;
+console.log(MONGODB_URI);
+
+let app = express();
+
+let leagueRouter = require('./routes/fantasy-routes');
+
+app.use(leagueRouter);
+app.use(jsonParser);
+
+app.get('/', (req, res) => res.json({msg: 'hello'}));
+
+mongoose.Promise = Promise; //uses promises with mongoose.
```

Comment:
A likely suggestion here in to utilize the Bluebird library for your promise implementation. It's going to provide a better API than the native Node implementation.